src/shared/qr.ts

```diff
@@ -245,3 +254,28 @@ export const getNewOutbreakExposures = (
   const newOutbreakExposures = detectedExposures.filter(item => newIds.indexOf(item.outbreakId) > -1);
   return newOutbreakExposures;
 };
+
+/**
+ * Look at all outbreak/checkin matches and remove duplicate matches
```

Comment:
By duplicate, this means outbreaks that have the same time stamp right? If I scan the same outbreak within a minute of each other I should see two outbreaks in my recent exposure?

Reply:
Yes 2 scans that fall in the same outbreak should result in 2 exposures